examples/eulerflow_profiling.py

```diff
@@ -0,0 +1,229 @@
+__copyright__ = """
```

Comment:
Do you mean to have an Euler file here?

Reply:
This was just due to merging with the other array context branch I believe. Removed.